spec/features/step_by_step_actions_spec.rb

```diff
@@ -21,11 +21,34 @@
     end
   end
 
+  context "Step by step has been submitted for 2i" do
+    before do
+      given_there_is_a_step_by_step_with_status "submitted_for_2i"
+    end
+
+    scenario "show the relevant actions to the step by step author" do
+      when_I_visit_the_step_by_step_page
+      then_there_should_be_no_primary_action
+      and_the_secondary_action_should_be "Preview"
+      and_there_should_be_tertiary_actions_to %w(Delete)
+    end
+  end
+
+  def given_there_is_a_step_by_step_with_status(status)
+    @review_requester = create(:user, name: "Original Author")
+    @reviewer = create(:user, name: "2i reviewer")
```

Comment:
This line isn't used yet (but agree it will be useful when we continue adding 2i features). I'd be tempted to leave it out for now.

Reply:
I tried without it and I get
`ActiveRecord::RecordInvalid:
  Validation failed: Status submitted_for_2i, requires a draft and review_requester_id to be present`

Reply:
It needs `@review_requester` but it shouldn't need `@reviewer` 🤔 can I just confirm you tried deleting line 39?

Reply:
Thanks for helping with this @ChrisBAshton, as it seems along with removing `@reviewer` I remove the `review_requester_id` too. Now it's all good, tests are passing.